CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.4.1
```

Comment:
Shouldn't it be 1.4.2? Doesn't match with the version in VERSION file.

Reply:
good catch...